ee/codegen/src/generators/nodes/bases/base.ts

```diff
@@ -518,7 +518,8 @@ export abstract class BaseNode<
   protected getNodeTrigger(): AstNode | undefined {
     if (
       this.nodeData.trigger &&
-      this.nodeData.trigger.mergeBehavior !== "AWAIT_ATTRIBUTES"
+      this.nodeData.trigger.mergeBehavior !== "AWAIT_ATTRIBUTES" &&
+      this.nodeData.trigger.mergeBehavior !== "AWAIT_ANY"
```

Comment:
we do _not_ want this here. Only for the nodes in the displayable directory that default to AWAIT_ANY such as `src/vellum/workflows/nodes/displayable/bases/api_node/node.py` and `src/vellum/workflows/nodes/displayable/bases/inline_prompt_node/node.py`